src/protojure/grpc/client/utils.clj

```diff
@@ -16,7 +16,7 @@
 (defn- put [ch val]
   (p/promise
    (fn [resolve reject]
-     (async/put! ch val resolve))))
+     (when val (async/put! ch val resolve)))))
```

Comment:
I think this will break when val == nil.  Suggest instead to do something such as:

```
(if (some? val)
   (async/put! ch val resolve)
   (resolve true))
```